airflow/kubernetes/pre_7_4_0_compatibility/pod_generator.py

```diff
@@ -72,7 +72,7 @@ def rand_str(num):
     return "".join(secrets.choice(alphanum_lower) for _ in range(num))
 
 
-def add_pod_suffix(pod_name: str, rand_len: int = 8, max_len: int = 80) -> str:
+def add_pod_suffix(pod_name: str, rand_len: int = 8, max_len: int = 63) -> str:
```

Comment:
Do we want to change it here? This just for pre-7.4.0 compatibility it seems?

Reply:
Okay, I'll keep that file unchanged ;)